src/components/RequestAmount/index.js

```diff
@@ -81,35 +89,49 @@ export class RequestAmount extends PureComponent<Props> {
     }
   }
 
-  render() {
-    const { t, value, account, rightUnit, getCounterValue } = this.props
+  renderInputs(containerProps: Object) {
```

Comment:
i would probably recommend `*`  over `Object`, even tho it's probably infered to `any`. best would be to specify the type